xdsl/tools/xdsl_run.py

```diff
@@ -62,6 +62,7 @@ def register_all_arguments(self, arg_parser: argparse.ArgumentParser):
         arg_parser.add_argument(
             "--index-bitwidth",
             choices=(32, 64),
+            default=Interpreter.DEFAULT_BITWIDTH,
```

Comment:
This is defined as the system bitwidth anyway

Reply:
Not clear to me what happened before if this wasn't specified

Reply:
I added a comment, the argument value is None by default, so it was assigning None explicitly in the init, which was propagated